src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java

```diff
@@ -48,7 +48,7 @@ public void toModelType_validPersonDetails_returnsPerson() throws Exception {
     public void toModelType_invalidName_throwsIllegalValueException() {
         JsonAdaptedPerson person =
                 new JsonAdaptedPerson(INVALID_NAME, VALID_PHONE, VALID_EMAIL, VALID_ADDRESS, VALID_DEADLINE,
-                        new ArrayList<>(VALID_NOTES.value), VALID_TAGS, VALID_FAVOURITE);
+                        VALID_NOTES.value, VALID_TAGS, VALID_FAVOURITE);
```

Comment:
I think it might be better to let `VALID_NOTES` just be `BENSON.getNotes().value` for standardization purposes! 😄  This is similar to how tags are represented under `VALID_TAGS` (as a `List`).

Reply:
You're right! Changing it now.